fenix/app/src/main/java/org/mozilla/fenix/datastore/DebugSettingsDataStore.kt

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.datastore
```

Comment:
💡 This could be in the `debugdrawer.data` package since it's related to that feature and other packages don't need to access it.

Reply:
made this change over in https://github.com/mozilla-mobile/firefox-android/pull/4520